reviews/views.py

```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from users.utils import login_required
+from .models import Review
+
+class ReviewView(View):
+    @login_required
+    def post(self,request):
+        try:
+            data    = json.loads(request.body)
+            product = data['product']
```

Comment:
`product_id` 로 명확하게 받아주세요!
```suggestion
            product_id = data['product_id']
```